### DISCUSSION TITLE
How shall I get "KovanETH" on Kovan Network?

### DISCUSSION THREAD

**Discussion by frank0532** [smartcontractkit/full-blockchain-solidity-course-py#1686] [General]
I have requested some from this "https://gitter.im/kovan-testnet/faucet#", but many days later, I still receive None.
Is there another method to get this? I want to test my code  reccently and my address is  "0xd22f0009EE02B0abac367d7F7E2Be968b254295e" and shall I get a little?

**Comment by cromewar** [comment]
Hello @frank0532 
Yeah, those faucets have limited amounts and are not working time to time.
Please use the Chainlink Oficial Faucet: https://faucets.chain.link/
Also, please take in consideration that the ETH merge is coming so the kovan and rinkeby networks will be useless, try using Goerli instead.
